processmanager/manager.go

```diff
@@ -73,16 +73,9 @@ var (
 // fileIDMapper and symbolReporter. Specify nil for fileIDMapper to use the default
 // implementation.
```

Comment:
This comment needs to be updated or removed as it's -partly- no longer relevant.